### PR TITLE
Create RunSettingsArgs.md to documents addition argument

### PR DESCRIPTION
**Pull request by Faizan2304**

### CODE REVIEW

**Comment by sbaid** [comment]
cc @pvlakshm pl take a read

File: docs/RunSettingsArgs.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  This argument will help you to specify runsettings settings through commandline arguments. For example, you can change the .NET Framework on which the tests will be run, the directory where test results are delivered during a test run.
+  
+  **Note:** Runsettings is a *.runsettings file which allow user to configure unit test. More info here :  https://msdn.microsoft.com/en-us/library/jj635153.aspx 
+  
+  ## Syntax
+  
+  `vstest.console.exe [Arguments] [Options] [[--] <args>...]]` or 
+  `dotnet test [options] <PROJECT> [[--] <additional arguments>...]]`
+  
+  `args` or `additional arguments` may be specified as name-value pair of the form `n=v`, where `n` is the argument name, and `v` is the argument value. All arguments after `--` will be treated as `additional arguments`. Use a space to separate multiple arguments.
```
Comment (sbaid) [inline docs/RunSettingsArgs.md]: pl change args to additional arguments in vstest.console as well for consistency.

File: docs/AdditionalArguments.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Passing additional arguments through commandline
+You are here because you are looking for syntax & details to pass additional arguments to either `vstest.console.exe` or `dotnet test` 
+
+Additional arguments is a way to pass arguments as [name]=[value] with a separator `-- `. Note the space after --. This can be used to pass additional arguments to test extensions like test adapters. They will be available to the test extensions as part of `runsettings`.
```
Comment (harshjain2) [inline docs/AdditionalArguments.md]: ``They will be available to the test extensions as part of `runsettings`. ``
Updated arguments is available to all who uses runsettings and not only to extensions.

File: docs/AdditionalArguments.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Passing additional arguments through commandline
```
Comment (harshjain2) [inline docs/AdditionalArguments.md]: `additional arguments`
This is not really additional arguments, it is something equivalent to specifying runsettings through CLI.
Consider changing it to RunSettings arguments

Comment (harshjain2) [inline docs/AdditionalArguments.md]: Update at other places as well.

File: docs/AdditionalArguments.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  <MSTest>  
+    <MapInconclusiveToFailed>True</MapInconclusiveToFailed>  
+  </MSTest>   
+</RunSettings> 
+```
+```
+dotnet test --settings additionalargs.runsettings
+```
+  
+This syntax can be used to pass any additional arguments to test framework adapters or extensions like logger through the CLI.
```
Comment (harshjain2) [inline docs/AdditionalArguments.md]: Suggestion : This line is redundant with line no 4

File: docs/RunSettingsArguments.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Passing runsettings arguments through commandline
+You are here because you are looking for syntax & details to pass runsettings configurations to either `vstest.console.exe` or `dotnet test` through commandline.
+
+`runSettings arguments` are used to add/update specific runsettings configurations. The updated runsettings configurations will be available to `TestPlatform` and `test extensions` (E.g. test adapter, etc.) through runsettings.
+
+`runSettings arguments` may be specified as name-value pair of the form `[name]=[value]` after `-- `. Note the space after --. Use a space to separate multiple `[name]=[value]`.
```
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: suggestion :  `may be` -> `can be`

File: docs/RunSettingsArguments.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# Passing runsettings arguments through commandline
+You are here because you are looking for syntax & details to pass runsettings configurations to either `vstest.console.exe` or `dotnet test` through commandline.
+
+`runSettings arguments` are used to add/update specific runsettings configurations. The updated runsettings configurations will be available to `TestPlatform` and `test extensions` (E.g. test adapter, etc.) through runsettings.
```
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: suggestion: runsettings configurations -> `runsettings configurations`

File: docs/RunSettingsArguments.md (hunk 5 above)
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: Suggestion : Can add example for specifying multiple `[name]=[value]` pairs.

File: docs/RunSettingsArguments.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+`runSettings arguments` may be specified as name-value pair of the form `[name]=[value]` after `-- `. Note the space after --. Use a space to separate multiple `[name]=[value]`.
+
+
+For example, passing argument `-- MSTest.MapInconclusiveToFailed=True` in (1) below is equivalent to passing argument `--settings additionalargs.runsettings` in (2) below.
```
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: Suggestion : Move `--settings` to new line.

File: docs/RunSettingsArgs.md (hunk 1 above)
Comment (Faizan2304) [inline docs/RunSettingsArgs.md]: Sure

File: docs/RunSettingsArguments.md (hunk 6 above)
Comment (Faizan2304) [inline docs/RunSettingsArguments.md]: Fixed!!

File: docs/RunSettingsArguments.md (hunk 5 above)
Comment (Faizan2304) [inline docs/RunSettingsArguments.md]: In next commit

**Pull request merged**